lib/supavisor.ex

```diff
@@ -16,19 +16,19 @@ defmodule Supavisor do
 
   @registry Supavisor.Registry.Tenants
 
-  @spec start_dist(id, secrets, Node.t() | boolean()) ::
+  @spec start_dist(id, secrets, atom(), Node.t() | boolean()) ::
           {:ok, pid()} | {:error, any()}
-  def start_dist(id, secrets, force_node \\ false) do
+  def start_dist(id, secrets, log_level \\ nil, force_node \\ false) do
```

Comment:
function with multiple defaults are sometimes confusing to use and are hard to grow in the future. It's fine to leave this as is but you can also bite the bullet right now and convert to options.

```suggestion
  def start_dist(id, secrets, options \\ []) do
    options = Keyword.validate!(options, log_level: nil, force_node: false)
    log_level = Keyword.fetch!(options, :log_level)
    force_node = Keyword.fetch!(options, :force_node)
```